libs/adsp-feedback/src/main.ts

```diff
@@ -463,6 +463,9 @@ class AdspFeedback implements AdspFeedbackApi {
     if (this.radio2Ref.value && this.radio2Ref.value.checked === true) {
       this.sendButtonRef.value?.removeAttribute('disabled');
     }
+    if (this.radio2Ref.value?.checked === false && this.radio1Ref.value?.checked === true) {
+      this.sendButtonRef.value?.removeAttribute('disabled');
+    }
```

Comment:
Could you please add some unit tests to check your changes?  In general, when working on the code, it is best practice to add unit tests to ensure your changes haven't broken anything. (I know, the original author should have added tests, but lets start doing things properly).

Reply:
Added new test cases to ensure latest code changes haven't broken any existing functionality.